src/docs/design/project-identity.mdx

```diff
@@ -1,5 +1,5 @@
 import { Divider, ImageGrid, MaterialDesignDescription } from '../../app/components';
-import PxblueSmallAltIcon from '@pxblue/icons-mui/PxblueSmallAlt';
+import PxblueSmallAltIcon from '@brightlayer-ui/icons-mui/PxblueSmallAlt';
```

Comment:
PxblueSmallAlt reference.

Reply:
PxblueSmallAlt will be replaced by new Blui icon once it's been created